components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/OrganizationManagerImpl.java

```diff
@@ -332,8 +332,9 @@ public void deleteOrganization(String organizationId) throws OrganizationManagem
                 }
             }
         }
+        int organizationDepthInHierarchy = organizationManagementDAO.getOrganizationDepthInHierarchy(organizationId);
```

Comment:
Are we using this variable in any other places rather than 337? If not we can directly get the value to 337 right?

Reply:
addressed through following PR
https://github.com/wso2-extensions/identity-organization-management-core/pull/67

Reply:
This PR was not merged with the above changes since reading the `organizationDepthInHierarchy` after the deletion of the organization causes to return a -1 value each time. This causes misbehaviour in tenant deletion flow.